### PR TITLE
manual: enhanced tagging view desc with item lists

### PR DESCRIPTION
**Pull request by VasylTretiakov**
added item lists for view characteristics and tag status indicator

### CODE REVIEW

File: doc/usermanual/lighttable/panels/tagging.xml [hunk 1]
```diff
@@ -116,21 +116,36 @@
...
-      <para>
-        On the right the count of images attached to the tag, in the database for dictionary view or
-        among the selected images for attached view.
-      </para>
-
+        
+        <itemizedlist>
+          
+          <listitem><para>
+            On the left there is an indicator giving the status of the tag in relation with the selected images:
```
Comment (Balise42) [inline doc/usermanual/lighttable/panels/tagging.xml]: On the left, an indicator gives the status of the tag [...]

File: doc/usermanual/lighttable/panels/tagging.xml [hunk 2]
```diff
@@ -116,21 +116,36 @@
...
+            
+            <itemizedlist>
+              
+              <listitem><para>
+                A <quote>check-mark</quote> means the tag is attached to all selected images.
```
Comment (Balise42) [inline doc/usermanual/lighttable/panels/tagging.xml]: means that the tag

File: doc/usermanual/lighttable/panels/tagging.xml [hunk 3]
```diff
@@ -116,21 +116,36 @@
...
+              </para></listitem>
+              
+              <listitem><para>
+                A <quote>minus</quote> means that the tag is attached at least to one selected image.
```
Comment (Balise42) [inline doc/usermanual/lighttable/panels/tagging.xml]: is attached to at least one selected image?

File: doc/usermanual/lighttable/panels/tagging.xml [hunk 4]
```diff
@@ -116,21 +116,36 @@
...
+                On tree view, on the parent node, it means that at least one child tag is attached to
+                a selected image.
+              </para></listitem>
+              
+            </itemizedlist>
+          </para></listitem>
+
+          <listitem><para>
+            On the right the count of images attached to the tag, in the database for dictionary view or
```
Comment (Balise42) [inline doc/usermanual/lighttable/panels/tagging.xml]: The number on the right is the count of images attached to the tag, whether in the database for the dictionary view, or among the selected images for the attached view.

Maybe. I'm not convinced by the "whether... or" syntax, but for sure this sentence lacks a verb :)

Comment (VasylTretiakov) [reply doc/usermanual/lighttable/panels/tagging.xml]: I reworked the whole thing, please check

File: doc/usermanual/lighttable/panels/tagging.xml [hunk 5]
```diff
@@ -115,22 +115,43 @@
...
+                Absence of any marking in the tag dictionary view means that the tag exists in the database, 
+                but is not attached to any of the selected images. In tree view, for a parent tag it also means
+                that none of its child tags is attached to any of the selected images.
+              </para></listitem>
+              
+            </itemizedlist>
+          </para></listitem>
+
+          <listitem><para>
+            The number in parentheses on the right from the tag is the attached image count. The attached tags
```
Comment (Balise42) [inline doc/usermanual/lighttable/panels/tagging.xml]: I think it's "right of" (but I'm unsure). And then I don't really like "considers" - I'd go for "In the attached tags view, this count refers to the selected images; in the tag dictionary views, it refers to the whole database." Or something like that.

Comment (VasylTretiakov) [reply doc/usermanual/lighttable/panels/tagging.xml]: rephrased

**Comment by TurboGit** [comment]
Thanks !

**Pull request merged**